### ISSUE TITLE
Error "parserOptions.operations need to be set and loaded"

### ISSUE THREAD

**Issue by mhatvan** [dotansimha/graphql-eslint#577]
**Describe the bug**
We want to use e.g. `@graphql-eslint/no-unused-fields` rule but it complains and breaks linting: 

`Error while loading rule '@graphql-eslint/no-unused-fields': Rule 'no-unused-fields' requires 'parserOptions.operations' to be set and loaded.`

To my understanding and after reading the explanatory link that comes with the error message, setting operations like `"operations": ["./{apps,libs,pages}/**/*.graphql"],` should be fine.

Before, it was actually set as `"operations": ["./**/*.graphql"],` but that didn't help.

**To Reproduce**

Here is a gist of the eslint config: https://gist.github.com/mhatvan/d3ee11e8f3a3ae19e277b9579ee82e07

Sorry for the big file, config is further below.

**Environment:**

- OS: Ubuntu 21.04
- `@graphql-eslint/...`: v2.0.0
- NodeJS: v14.17.2

**Comment by mhatvan** [comment]
Both globs don't work, any clue where I could start investigating?

**Comment by ardatan** [comment]
Could you create a reproduction on CodeSandbox?

**Comment by mhatvan** [comment]
Will provide one soon.

**Comment by mhatvan** [comment]
I created a slimmed-down variant locally, but was unable to reproduce the issue. All rules work properly.

Hard to track down since our application is 50+ packages in a monorepo. Will investigate further at some point.

**Comment by B2o5T** [comment]
It’s hard to debug without reproduction repo (I used `graphql-eslint` already in monorepo project and I didn't have any problems), maybe [this comment](https://github.com/dotansimha/graphql-eslint/discussions/456#discussioncomment-785112) will help you.

**Comment by beaucollins** [comment]
@mhatvan does your monorepo potentially load up different rulesets for different GraphQL schemas?

I'm seeing a problem where a I have a different set of overrides for a code path that queries a third-party's graph. When I have both sections set up the `@graphql-eslint/eslint-plugin` seems to get confused.

```js
    {
      files: ['src/core/**/*.graphql'],
      parser: '@graphql-eslint/eslint-plugin',
      plugins: ['@graphql-eslint'],
      extends: ['plugin:@graphql-eslint/recommended'],
      parserOptions: {
        operations: ['src/core/**/*.graphql'],
        schema: './coreSchema.graphql',
      },
    },
    {
      files: ['src/third-party/**/*.graphql'],
      parser: '@graphql-eslint/eslint-plugin',
      plugins: ['@graphql-eslint'],
      extends: ['plugin:@graphql-eslint/recommended'],
      parserOptions: {
        operations: ['src/third-party/**/*.graphql'],
        schema: './node_modules/third-party/schema.graphql',
      },
    },
```

**Comment by mhatvan** [comment]
Hey @beaucollins, I am not working as a freelancer for the project mentioned above anymore, so can't help you there, unfortunately. In my case, we only had one schema set, not multiple ones though.
